docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/ecommerce/index.md

```diff
@@ -1,7 +1,20 @@
 ---
 title: "E-commerce"
 sidebar_position: 105
+hide_title: true
 ---
+# E-commerce Quickstart
+
+```mdx-code-block
+import Admonition from "@theme/Admonition";
+import {SnowplowIcon} from "@site/src/components/SnowplowIcon/SnowplowIcon";
+```
+
+<Admonition icon={SnowplowIcon()} title="Unleash the power of your behavioral data" type="tip">
```

Comment:
These admonitions all have the same text, save for the name of the accelerator. Could you make this a component, or at least a reusable markdown block? (Check the telemetry one for an example of a reusable block with parameters.) That way we wouldn’t need to change this in 6 different places when needed.

Reply:
I have done my best